modules/Bio/EnsEMBL/Gene.pm

```diff
@@ -1527,6 +1507,42 @@ sub havana_gene {
   return $ott;
 }
 
+=head2 biotype
+
+  Arg [1]    : Arg [1] : (optional) String - the biotype to set
+  Example    : my $biotype = $gene->biotype;
+               my $biotype = $gene->biotype('protin_coding');
+  Description: Returns the Biotype object of this gene.
+               When no biotype exists, defaults to 'protein_coding'.
+  Returntype : Bio::EnsEMBL::Biotype
```

Comment:
This is not returning an object in the default case.

Reply:
there is indeed the chance that a string is returned instead of a Biotype object.
That is when adaptor() does not exist in the gene or transcript object.
I'll change it to always return a biotype object, but will need to
use Bio::EnsEMBL::Biotype in the Gene and Trancript classes (to create a Biotype object there bypassing the BiotypeAdaptor).